mautic/mautic
# Allow to disable page tracking

| Q                                      | A
| -------------------------------------- | ---
| New feature/enhancement? (use the a.x branch)      | [X]
| Automated tests included?              | [X]  

#### Description:

Allow configuration option of a page to disable tracking of the given page.

1. adds config option to page config
2. the page won't get tracked

#### Steps to test this PR:

1. Create a common page in mautic
2. Visit the page in anonymous browser with the tracking disabled (available on landing page settings)
3. The visit should not appear in page details nor a new anonymous contact has to be created
4. The browser should receive no cookies
